### DISCUSSION TITLE
Is it possible to set the height of a textarea box?

### DISCUSSION THREAD

**Discussion by Daniel-Fei** [zauberzeug/nicegui#2817] [Q&A] [answered]
### Question

 I'm wondering if it's possible to set the height of a textarea box? I've tried adding the height or min-height properties in .classes(), but this method only increases the height of the background area below the input area, not the actual input area itself.

Looking forward to any suggestions, thank you!

**Comment by falkoschindler** [comment, marked as answer]
Hi @Daniel-Fei,

The input elements `ui.textarea`, `ui.input` and `ui.number` are based on [Quasar's QInput](https://quasar.dev/vue-components/input), which creates a rather complex DOM with an `<input>` tag somewhere inside. Changing style or classes only affects the outer most HTML element, which is not what you want. Instead you can use the "input-style" or "input-class" props to address the `<input>` tag:

```py
ui.textarea().props('input-class=h-64')
```

**Reply by Daniel-Fei** [reply]
It works! Thank you very much for your help!
